websites/docs-smoke-test/src/content/plan.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Plan Legal Agreements
+---
+
+# Terms and conditions
+
+TBD
```

Comment:
I'm not sure what test case this page represents.  "TBD" souhds like it's not ready or a leftover

Reply:
The idea here is to have a page similar to https://docs-kit.commercetools.vercel.app/docs-smoke-test/beta . It's used as sort of  dummy landing page for terms and conditions of the dummy plans defined in the smoke test site. Essentially when the user clicks on the `plan1` or `plan2` icons at the top of the pages, he will land on this page. TBD could be replaced with lorem ipsum or maybe I can just rephrase what I wrote here so that is self explanatory